ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -380,6 +398,23 @@ def _get_dtype_value1_value2_cov(
     return [dtype], value1, value2, rowVar, bias, ddof, fweights, aweights
 
 
+# higher_order_moment
+@st.composite
+def _higher_order_moment_data(draw):
+    shape = draw(helpers.get_shape(min_num_dims=2, max_num_dims=5))
+    order = draw(helpers.ints(min_value=0, max_value=10))
+    dtype, x = draw(
+        helpers.dtype_and_values(
+            available_dtypes=["float32", "float64"],
```

Comment:
Same question here.

Reply:
Hey @hello-fri-end , I revised this dtype issue and I added some `with_supported_dtypes `decorator to the functions that made the tests fail except for the paddle backend since it will be fixed in another PR